examples/js/renderers/SVGRenderer.js

```diff
@@ -118,10 +122,26 @@ THREE.SVGRenderer = function () {
 
 	}
 
+	function getSvgColor ( color, opacity ) {
+
+		var arg = Math.floor( color.r * 255 ) + ',' + Math.floor( color.g * 255 ) + ',' + Math.floor( color.b * 255 );
+
+		if ( opacity === undefined || opacity === 1 ) return 'rgb(' + arg + ')';
+
+		return 'rgba(' + arg + ',' + opacity + ')';
```

Comment:
By the way, similar approach can be used THREE.Color.getStyle(). 
One could add opacity parameter to the function and return rgba() string if opacity defined and not equal to 1